### PR TITLE
Support multi url pickers

### PR DESCRIPTION
**Pull request by bjarnef**
This PR add the following:

* Support for multiple picker
* Enable/disable pickers
* Localization

and some other minor changes in UI.

### CODE REVIEW

**Comment by kgiszewski** [comment]
@bjarnef So far so good. What is the plan (as far as install handling) of localization terms?

I'm testing on 7.2.8 so I'm curious if your localization doesn't work due to that. That could prove to be a breaking change.

screen shot 2016-04-12 at 1 29 27 pm

I have to test out the PVC yet.

**Comment by bjarnef** [comment]
@kgiszewski yeah, the support of localization in a plugin lang folder was first supported from 7.3.x, pre 7.3 version that use ```localize```directive and ```localizationService``` needed to merge the keys into the core language files.

Some package like Merchello and Analytics has an installer that merge language keys into core files if pre Umbraco 7.3 versions.

In Angular controller if could use a fallback if the lookup key was empty or doesn't exists, but I don't think you can do that in the views as far as I know.

Maybe you could take a look at the .cs files here: https://github.com/warrenbuckley/Analytics/tree/develop/Analytics

https://github.com/warrenbuckley/Analytics/blob/develop/Analytics/InstallHelpers.cs#L15-L21

https://github.com/warrenbuckley/Analytics/blob/develop/Analytics/TranslationHelper.cs

**Comment by bjarnef** [comment]
This might be useful to only run it for pre Umbraco 7.3 versions: https://github.com/warrenbuckley/Analytics/blob/develop/Analytics/AnalyticsHelpers.cs#L16-L18

**Comment by kgiszewski** [comment]
@bjarnef I'm aware of the differences :) In Archetype we've avoided this issue by using our own directive. 

I'm not going to be able to have the time to add that functionality in.

**Comment by bjarnef** [comment]
Okay, but I think it is fine just using the core localize directive and localizationService.
Maybe just make a note about the breaking changes.. and if people use the new version and an "old" version of Umbraco 7, they could always manually merge the keys from the /lang folder :)

File: urlpicker/config/package.nuspec [hunk 1]
```diff
@@ -11,4 +11,4 @@
         <dependency id="UmbracoCms.Core" version="7.1.6" />
```
Comment (kgiszewski) [inline urlpicker/config/package.nuspec]: Looks like we need to change this to 7.3.0? Not sure the exact version number.

**Comment by kgiszewski** [comment]
Looks like the Nuget Dependency is set to 7.1.6. Let's move that to 7.3.0 and I don't see that the project is built against 7.3.

**Comment by kgiszewski** [comment]
Looks like the core version built is still 7.1.1: https://github.com/imulus/uWestFest/blob/master/urlpicker/src/UrlPicker.Umbraco/packages.config#L23

**Comment by bjarnef** [comment]
I have just kept the references as it was, but feel free to update them if is should be target against Umbraco 7.3.x..  The option the add a language folder in package folder in App_Plugins was added in 7.3.0

**Comment by kgiszewski** [comment]
I'm going to have to ponder this a bit more. Because if we depend on 7.3.x, any Nuget users will get an unpleasant surprise of auto-upgrading to 7.3.x core if they are using anything older. There are massive breaking changes in 7.3 for some installs that I currently maintain that would mean I can't use this update on that install. So perhaps not bumping the version number is best.

**Comment by kgiszewski** [comment]
@bjarnef Thanks again for this. I'll be pushing this out soon as a release.

**Pull request merged**